packages/apps/human-app/server/src/integrations/reputation-oracle.service.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { HttpService } from '@nestjs/axios';
+import { lastValueFrom } from 'rxjs';
+import { SignupWorkerDto } from '../interfaces/signup-worker-request.dto';
+
+@Injectable()
+export class ReputationOracleService {
+  constructor(
+    private configService: ConfigService,
+    private httpService: HttpService,
+  ) {}
+
+  async signupWorker(signupWorkerDto: SignupWorkerDto): Promise<void> {
+    try {
+      const method = 'POST';
+      const baseUrl = this.configService.get<string>('REPUTATION_ORACLE_URL');
```

Comment:
Make separate Environment Variables service with default values handling

Reply:
Fixed in commit 2f386e85